prymer/primer3/primer3_parameters.py

```diff
@@ -93,6 +135,26 @@ class PrimerAndAmpliconParameters:
         primer_max_3p_homodimer_tm: the max melting temperature acceptable for self-complementarity
             anchored at the 3' end
         primer_max_hairpin_tm: the max melting temperature acceptable for secondary structure
+        product_size_lt: weight for products shorter than
+            `PrimerAndAmpliconParameters.amplicon_sizes.opt`
+        product_size_gt: weight for products longer than
+            `PrimerAndAmpliconParameters.amplicon_sizes.opt`
```

Comment:
The frequency with which these come in pairs makes me think maybe we'd want to have a `WeightPair` (better name needed) class like `MinOptMax` so that you could do:

```python
amplicon_size_wts = Weights(0.5, 1.5)
```

Reply:
Also two other thoughts:
- I think the weights should match naming with above, so e.g. where we've said "amplicon_tms" we should use "amplicon" instead of "product" in the weights (or vice versa).
- I think for clarity all weights should either have a `wt_` prefix or `_wt` suffix; some do and some don't here.